cdap-app-fabric/src/test/java/co/cask/cdap/internal/profile/ProfileMetadataTest.java

```diff
@@ -43,100 +44,105 @@
 
   @Test
   public void testProfileMetadata() throws Exception {
+    MetadataSubscriberService metadataSubscriberService = getInjector().getInstance(MetadataSubscriberService.class);
```

Comment:
start it in a BeforeClass method and stop it in an AfterClass method. Otherwise, you won't be able to add any other test methods to this class.